### PR TITLE
feat: new or updated contact in google

### PR DESCRIPTION
**Pull request by abuaboud**

### CODE REVIEW

File: packages/frontend/src/app/modules/flow-builder/store/builder/flow-item-details/flow-items-details.effects.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class FlowItemsDetailsEffects {
 						return new FlowItemDetails(
 							TriggerType.PIECE,
 							c.displayName,
-							`Trigger this flow following a specific event on ${c.displayName}`,
+							``,
```
Comment (AbdulTheActivePiecer) [inline packages/frontend/src/app/modules/flow-builder/store/builder/flow-item-details/flow-items-details.effects.ts:55]: Please bring back the description, the UI will not look good and feel like something is missing.

File: packages/pieces/src/lib/apps/google-contacts/trigger/new-contact.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    description: 'Triggers when there is a new or updated contact',
+    props: {
+        authentication: googleContactsCommon.authentication,
+    },
+    sampleData: {
+    },
+    type: TriggerStrategy.POLLING,
+    async onEnable(ctx) {
+        const response = await listContacts(getAccessTokenOrThrow(ctx.propsValue['authentication']), undefined);
+        console.log(`Saved sync token ${response.body.nextSyncToken}`);
```
Comment (AbdulTheActivePiecer) [inline packages/pieces/src/lib/apps/google-contacts/trigger/new-contact.ts]: Remove this.

**Comment by AbdulTheActivePiecer** [comment]
This trigger works when you delete, add or update, is it possible to separate them ?

File: packages/frontend/src/app/modules/flow-builder/store/builder/flow-item-details/flow-items-details.effects.ts (hunk 1 above)
Comment (abuaboud) [reply packages/frontend/src/app/modules/flow-builder/store/builder/flow-item-details/flow-items-details.effects.ts:55]: discussed offline

**Comment by abuaboud** [comment]
Addressed comments, merged

**Pull request merged**